.markdownlint.json

```diff
@@ -1,6 +1,6 @@
 {
   "default": true,
   "ignore_front_matter": true,
-  "MD013": { "line_length": 80, "code_blocks": false, "tables": false },
+  "MD013": { "line_length": 110, "code_blocks": false, "tables": false },
```

Comment:
Yes, I recommend that as weel. See first row of table in [comment above](https://github.com/betterscientificsoftware/bssw.io/pull/745#issuecomment-804480235). But, there is a question about what to do about pre-existing content.

Reply:
Whoops...I mis-understood. You are arguing for **NOT** enforcing line length. I think we should. The whole group needs to decide what to do about it.

Reply:
My view is that for raw text documents where the primary viewing is by some formatted output, then how you format the raw text should be dominated by the version control considerations (and use text wrapping in your editor for long lines).  So for Latex `*.tex` files, this is pretty straightforward since the raw text is not meant to be read, only edited and modified so you should keep one sentence per line.  But for markdown formats where the raw text is meant to be human readable, the choice is not so clear.  If you expect people to looking at and reading the raw text a lot, then you want to be formatting the lines (i.e. to 80 chars or so wide using emacs `Alt+Q` for eample) and keep line lengths down and nice square paragraphs.   Therefore, I would leave the issue of line lengths up to the authors of a given `*.md` if to enforce line length file if possible.

But since I don't see there being a lot of concurrent editing of `*.md` files in the bssw.io repo happening, merging considerations (where you want to keep one sentence per line) are likely not important so I guess enforcing line length and asking people to auto-format paragraphs is likely a good idea.  (But I have seen cases where section titles had to all be on one line and for longer text enforcing a line length would be a problem.)